.eslintrc

```diff
@@ -1,6 +1,11 @@
 {
   "root": true,
   "extends": "@1stg",
+  "rules": {
+    "no-undef": "off",
```

Comment:
Is this rule related to globals?
Would it make sense to add the globals so this could be re-enabled?

Reply:
I don’t remember why it was turned off. This is a draft for a reason. I expect it to be slightly messy in the beginning (docs are missing for example). This was just annoying at some point. I will re-evaluate ESLint rules to turn on when everything is more polished.